client/src/components/RuleCollectionBuilder.vue

```diff
@@ -1008,29 +1024,37 @@ export default {
             const requiresName =
                 buildingCollection && this.elementsType != "collection_contents" && !mappingAsDict.collection_name;
 
+            this.emptyValidityErrorMessages;
```

Comment:
Typo here, I think this was intended to call the method?

As an aside, this still mutates local component state in a computed and probably needs refactoring.